src/cheri/xycheriot.adoc

```diff
@@ -0,0 +1,678 @@
+ABI names for registers were similarly differeniated,
+using a prefix of `c` when the full capability was intended (`csp`, `ca0`, &c).
+{cheri_base_ext_name} does away with the distinction:
+all register selectors are `xN` and all ABI names are un-prefixed.
+
+==== `CGetAddr` and `CSub` Replacement
+
+CHERIoTv1 inherited from University CHERI and subsequently offered
+a `CGetAddr` instruction which copied a capability while
+explicitly zeroing its {ctag} and metadata, preserving only the address.
+`CGetAddr` largely dates back to early CHERI research machines,
+when capability registers were, like floating registers (without Zfinx),
+distinct from integer registers,
+and was largely obsolesced by CHERIoTv1 using a merged register file.
+Since {cheri_base_ext_name} likewise uses a merged regiser file,
+instructions consuming XLEN-width values use the
+XLEN-width address field of a capability while ignoring its {ctag} and metadata,
+and instructions producing XLEN-width results
+write capability values with zero {ctag} and metadata.
+Thus, `addi x1, x2, 0` exactly replaces CHERIoTv1's `CGetAddr x1, c2`;
+the latter form, and/or a `YADDRR` mnemonic consistent with <<SCADDR>>,
+may still be usefully provided by an assembler for documeting programmer intent.
+
+Similarly, CHERIoTv1's `CSub rd, cs1, cs2` instruction for computing
+the (integer) differences between two capabilities' address fields,
+may be replaced with integer subtraction, `sub rd, rs1, rs2`.
+
+==== Permission Bitfield Semantics
+
+The permissions bit field used by <<GCPERM>> (replacing `CGetPerm`)
+is permuted with respect to CHERIoTv1, and permissions are renamed.
+Nevertheless, permission *semantics* are in direct correspondence, thus:
+
+|===
+| CHERIoTv1 | {cheriot_ext_name}    |
+| EX        | <<x_perm>>            |
+| SR        | <<asr_perm>>          |
+| SE        | <<zyseal_se_perm>>    |
+| US        | <<zyseal_us_perm>>    |
+| U0        | SDP bit 0             |
+| GL        | <<zylevels1_gl_perm>> |
+| SL        | <<zylevels1_sl_perm>> |
+| LM        | <<lm_perm>>           |
+| LG        | <<zylevels1_lg_perm>> |
+| MC        | <<c_perm>>            |
+| SD        | <<w_perm>>            |
+| LD        | <<r_perm>>            |
+|===
+
+The bitwise AND behavior of CHERIoTv1's `CAndPerm` has been replaced with
+the bitwise AND NOT behavior of <<CLRPERM>>.
+Permission masks should thus be permuted and bitwise negated.
+
+==== Elimination of Special Capability Registers
+
+CHERIoTv1 inherited from University CHERI a notion of
+"special capability registers" (SCRs; its §7.10) in addition to CSRs.
+The base {cheri_base_ext_name} ISA has instead extended some CSRs to YLEN.
+CHERIoTv1's `MTCC`, `MScratchC`, and `MEPCC` SCRs directly correspond with
+{cheri_base_ext_name}'s <<mtvec>>, <<mscratch>>, and <<mepc>> YLEN-wide CSRs.
+CHERIoTv1 additionally had a `MTDC` SCR, which has proven to be redundant,
```

Comment:
Hmm, I don't think you should state that here, especially with all the discussions we are having to preserve MTDC anyway

Reply:
"had a `MTDC` SCR, which may yet make an appearance in {cheriot_ext_name}", perhaps?